README.md

```diff
@@ -99,6 +99,50 @@ stake_test1urmd9uh08pen8c26a2fn86weprjh52638mrdwc5gfac2u2s25zpat%
 ```
 </details>
 
+<details>
+  <summary>How to inspect address</summary>
+
+```console
+$ echo addr_test1vp2fg770ddmqxxduasjsas39l5wwvwa04nj8ud95fde7f7guscp6v | cardano-address address inspect
+{
+    "stake_reference": "none",
+    "spending_key_hash_bech32": "addr_vkh12j28hnmtwcp3n08vy58vyf0arnnrhtavu3lrfdztw0j0jng3d6v",
+    "address_style": "Shelley",
+    "spending_key_hash": "54947bcf6b760319bcec250ec225fd1ce63baface47e34b44b73e4f9",
+    "network_tag": 0,
+    "address_type": 6
+}
+
+$ echo addr_test1qp2fg770ddmqxxduasjsas39l5wwvwa04nj8ud95fde7f70k6tew7wrnx0s4465nx05ajz890g44z0kx6a3gsnms4c4qq8ve0n | cardano-address address inspect
+{
+    "stake_reference": "by value",
+    "stake_key_hash_bech32": "stake_vkh17mf09mecwve7zkh2jve7nkggu4azk5f7cmtk9zz0wzhz5efq2w6",
+    "stake_key_hash": "f6d2f2ef387333e15aea9333e9d908e57a2b513ec6d762884f70ae2a",
+    "spending_key_hash_bech32": "addr_vkh12j28hnmtwcp3n08vy58vyf0arnnrhtavu3lrfdztw0j0jng3d6v",
+    "address_style": "Shelley",
+    "spending_key_hash": "54947bcf6b760319bcec250ec225fd1ce63baface47e34b44b73e4f9",
+    "network_tag": 0,
+    "address_type": 0
+}
+```
+
+Details about possible address types are following.
+| address_type | binary prefix  |   Meaning                                                |
```

Comment:
Address type and binary prefix columns are the same number, just decimal/binary representations, right?

Can we add a reference to the shelley CDDL, if none present yet?